package.json

```diff
@@ -59,7 +59,7 @@
     "webpack-cli": "^6.0.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.15.1",
+    "@eppo/js-client-sdk-common": "/tmp/packages/js-sdk-common",
```

Comment:
This will update once [`js-sdk-common #296`](https://github.com/Eppo-exp/js-sdk-common/pull/296#pullrequestreview-3302099811) is merged